lib/plugins/ray_trace.js

```diff
@@ -23,6 +23,32 @@ function inject (bot) {
       }
     }
   }
+  const rayTraceEntity = (maxSteps = 256, vectorLength = 5 / 16) => {
+    const { height, position, yaw, pitch } = bot.entity
+    const cursor = position.offset(0, height, 0)
+    const x = -Math.sin(yaw) * Math.cos(pitch)
+    const y = Math.sin(pitch)
+    const z = -Math.cos(yaw) * Math.cos(pitch)
+
+    const step = new Vec3(x, y, z).scaled(vectorLength)
+
+    for (let i = 0; i < maxSteps; ++i) {
+      cursor.add(step)
+
+      // Checking for entity
+      for (const entity of Object.values(bot.entities)) {
+        if (entity === bot.entity) {
+          continue
+        }
+        const dist = entity.position.offset(0, entity.height / 2, 0).distanceSquared(cursor)
+        if ((dist <= 0.5) && (cursor.distanceTo(bot.entity.position.offset(0.5, 0.5, 0.5)) <= 5)) {
+          const sightEntity = entity
+          return sightEntity
```

Comment:
```suggestion
          return entity
```
whats the use in a variable here if its only going to be returned instantly